### PR TITLE
92 speaker card improved

### LINKED ISSUE #92: Improve Speaker Card
**Is your feature request related to a problem? Please describe.**
The speaker card doesn't show all the information we've collected yet. 

**Describe the solution you'd like**
Speaker card should display other fields such as
- Second job / association (one suggestion was this could be academic affiliation for PhDs)
- Biography
- Social links
- Photo?
- Pronouns after #91

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #92 
Experimenting with the card layout using `v-expansion-panels` to expand open the speaker bios.

Changes include:
- Click header to reveal speaker bio and social links
- Rounded language chips to differentiate from topic tags. Thoughts? 
- Added pronouns
- Adding some design guidelines now that we're including Font Awesome icons.
- Removed favoriting since so much stuff shifted. It was cluttering up the code anyway.

Remaining work:

- [x] Adjust fonts for pronouns
- [x] Allow a second job / affiliation (`secondary_title`, `secondary_affiliation`)
- [x] Test on mobile -- Responsive design adjustments
- [x] Add links to social profiles via icons (need to find an icon set matching material design)

Screenshots:

Mobile Card:
SpeakHer

Mobile Expanded Card:
SpeakHer

Desktop Card:
SpeakHer

Desktop Expanded Card:
SpeakHer

### CODE REVIEW

**Comment by tuttiq** [comment]
Thanks for the PR @ann-kilzer! I think the design looks a little weird as some elements look unaligned due to different margin/padding or wrapped line. Let me see if I can tweak a little on my side and get back to you on this 🙇

**Comment by ann-kilzer** [comment]
Let me poke around with the margins / padding a bit more. As I mentioned, the gridding needs more polish.

What do you think about the expand / collapse?

**Comment by ann-kilzer** [comment]
Okay, used CSS instead of the built-in classes to get better control over the gridding, and at least have vertical alignment working.

Can you annotate a picture with anything you'd liked adjusted?

One area for improvement is the email / contact button, which is tricky to adjust around the expansion arrow. We could consider moving it somewhere else, or rewriting the expansion panel for greater customization. However, it might also be nice to get a working version out and then iterate on that too.

**Comment by ann-kilzer** [comment]
An experiment in another direction, moving some things around:

- Make primary action (connect with speaker) a FAB
- Location aligned left
- drop pronouns into hidden section
- raise social links to header

Thoughts?

SpeakHer

**Comment by tuttiq** [comment]
Thank you so much for the alternatives, @ann-kilzer ! This last version looks pretty neat, I'd be fine shipping it like that for now and improving it iteratively whenever we find good tweaks 👍

**Comment by ann-kilzer** [comment]
Thanks for your feedback! I will recode that version in a less hacky way tomorrow.

File: web/src/components/speaker/Languages.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      required: false,
+      default: () => [],
+    },
+  },
+  computed: {
+    dynamicPadding() {
+      if (this.$vuetify.breakpoint.mdAndUp) {
+        return {
+
+        };
```
Comment (tuttiq) [inline web/src/components/speaker/Languages.vue:37]: Do we need the brackets and empty line here? maybe an empty string and semicolon would suffice?

File: web/src/components/speaker/CardFooter.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          v-if="$vuetify.breakpoint.smAndDown"
+          :facebook="speaker.get('facebook_url')"
+          :twitter="speaker.get('twitter_url')"
+          :linked-in="speaker.get('linkedin_url')"
+          :website="speaker.get('website_url')"
+        />
+        <v-spacer />
+        <v-btn
+          color="primary"
+          fab
```
Comment (tuttiq) [inline web/src/components/speaker/CardFooter.vue:44]: this is just awesome

File: README.md [hunk 3]
```diff
@@ -20,6 +20,17 @@ All contributors to this repository must follow the [Code of Conduct](https://ww
...
+
+1. We use Material Design via the [Vuetify](https://vuetifyjs.com/en/) framework for this site.
+
+2. Icons should be selected from the [Material Icon Set (Filled)](https://material.io/resources/icons/?style=baseline), with a few exceptions where social media icons may be taken from FontAwesome.
+
+3. Please ensure new features support Responsive Design (friendly to mobile, tablet, desktop, etc.).
+
+4. We love mockups and paper prototypes, so if you have a suggestion and can sketch your idea, that's helpful.
+
+Please ask @ann-kilzer if you have questions!
```
Comment (tuttiq) [inline README.md:33]: ❤️

**Review by tuttiq — APPROVED** [review]
Looks awesome!! Thank you so much for this!

File: web/src/components/speaker/Languages.vue (hunk 1 above)
Comment (ann-kilzer) [reply web/src/components/speaker/Languages.vue:37]: It seemed like the dynamic classes wanted an empty object, though I could neaten up the spacing. May I do that in a later PR?

**Pull request merged**